src/index.js

```diff
@@ -1 +1,80 @@
-console.log("Hello World!");
+import './style.css';
+
+// const gameID = 'X9fxHdLZsiAYP8qUTFeL';
+
+const gameID = 'SAafjYwuy9TE9mm0ILLg';
+const url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameID}/scores/`;
+// https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/SAafjYwuy9TE9mm0ILLg/scores/
```

Comment:
- [OPTIONAL]  Please kindly delete the comment line of code to have a good and clean codebase 🙃 🙃